README.md

```diff
@@ -1,5 +1,54 @@
 
-Currently almost nothing works except the basic login and chances are high, that I will never finish this.
+Currently, this project is in the early stages of development, and it is unclear, if it will ever be fully complete. The goal is of course to be 100% compatible with official game clients, and to offer all the features the official servers do. However, due to the nature of an actively updated game like *Shakes & Fidget*, this is a moving target. If you feel you can contribute to this project in any way, feel free to do so. Any help is appreciated, be it in the form of code, documentation, or even just testing and bug reporting.
+
+
+## Getting Started
+To run this project for yourself, you can simply clone this repository and build the server yourself, or check the [releases](https://github.com/the-marenga/sf-server/releases) page for pre-built binaries.
+
+### Connecting clients to your server
+Launching the server by itself will publish the server on `http://0.0.0.0:6768` by default. Only `HTTP` (without) TLS/SSL is supported by the Rust application itself, that becomes a challenge when you want to use the official *Shakes & Fidget* client, to connect. That is because it prefixes all server connections with *https://*. To overcome this, there are two possible options: For a quick and easy setup (for example as a development environment), you can use a **(I)** *Firefox* extension. For a more permanent solution, you can use **(II)** a reverse proxy like *nginx*, which will only require a one-time setup on the server side, unlike the add-on, which every user who wants to connect to play on the server will have to install. Both options are described in the following sections, and examples are provided inside this repository.
+
+In both cases you will need to override the `https://sfgame.net/config.json` response in your browser, such that it returns the address of your server as one of the (or if you wish: the only) server(s). This is necessary, for both options, and the only way around it would be to redistribute the client yourself, which we already strongly advised against. Instructions for overriding the `config.json` response can also be found in the respective section below.
+
+#### (I) Usage with Firefox extension
+In the [/extension](https://github.com/the-marenga/sf-server/tree/main/extension) directory of this repository, you can find a *Firefox* add-on, that effectively acts like a patch to the official client to no longer require `HTTPS`.
+
+For obvious reasons, this is not feasible for any production-esque environments, but it is a quick and easy way to try things out. Detailes instructions on how to install the extension can be found in the [README.md](https://github.com/the-marenga/sf-server/tree/main/extension/README.md) file in the same directory.
```

Comment:
Wait firefox can not save the overrides to files? Edge and Opera (I guess all Chromium based) do it by default :O 

But yea, I'll fix it, should have read your README instead of just scanning the extension code xx
My bad